src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -772,6 +766,79 @@ Value emitScalarOpFor<ONNXSignOp>(ConversionPatternRewriter &rewriter,
   return create.math.select(zeroPredicate, zero, plusSelect);
 }
 
+//===----------------------------------------------------------------------===//
+// Scalar unary ops for lowering ONNXErfOp
+//===----------------------------------------------------------------------===//
+template <>
+struct ScalarOp<ONNXErfOp> {
+  using FOp = CustomScalarOp;
+  using IOp = NotSuportedScalarOp;
+};
+
+template <>
+double analyzeSimdFor<ONNXErfOp>(Type t, int64_t &von, int64_t &son) {
+  return simdAnalysis(
+      {GenericOps::CompareGop, GenericOps::DivGop, GenericOps::FmaGop,
+          GenericOps::MulGop, GenericOps::SelectGop, GenericOps::AbsGop,
+          GenericOps::ExpGop},
+      {1, 1, 6, 4, 1, 1, 1}, t, von, son);
```

Comment:
`Fma` will be `Mul` and `Add` in case of IntegerType. How do we count them in such a situation?

@AlexandreEichenberger any though.

Reply:
@AlexandreEichenberger Thanks for the answer. 
I understand that we can simply count the operation number without considering the target type is integer or float.
Please let me know if my understanding is not correct.